src/OperationalInsights/OperationalInsights/OperationalInsights.csproj

```diff
@@ -12,7 +12,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.Management.OperationalInsights" Version="0.21.0-preview" />
-    <PackageReference Include="Microsoft.Azure.OperationalInsights" Version="0.10.0-preview" />
+    <PackageReference Include="Microsoft.Azure.OperationalInsights" Version="1.1.0" />
```

Comment:
Version also needs to be updated in the test csproj

Reply:
Done